tests/pck/023.0102.rounding.nobatch

```diff
@@ -0,0 +1,8 @@
+FV          
+RSI5B:12345678901A:1604
```

Comment:
I think you'll need to adjust the codacy check to ignore the example test files (they're not code but it seems to think they are and it's reporting syntax errors)

Reply:
Done